### PR TITLE
Week 4 sample solutions

### LINKED ISSUE #377: [Content Bug] Week 4 sample solutions need to be updated
**Identify Steps:**
* Update sample solutions in week 4 to match changes to the md files (removed onsubmit, jQuery, added event listeners, etc.)

**Week:**
4

## Describe the Issue

md file updated to reflect more modern coding conventions. Sample solutions will have to be updated also.

### PR DESCRIPTION
**Pull request by ganning127**
## Changes made
* Updated the sample solutions for the steps with updated event listeneres and such. 

Closes #377

### CODE REVIEW

File: .bit/tests/sample-solutions/week4/4.4/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
           <div class="c-well">
               <h1 id="h1">Bunnimage 🐇</h1>
                   <h2> Upload </h2>
-                  <form id="myform" onsubmit="getImage(event)" enctype="multipart/form-data">
+                  <form id="bunnForm" onsubmit="getImage(event)" enctype="multipart/form-data">
```
Comment (emsesc) [inline .bit/tests/sample-solutions/week4/4.4/index.html:18]: I think you missed this one

**Review by emsesc — CHANGES_REQUESTED** [review]
I think you missed one here, and if you're changing event listeners as well, you can update the rest of Week 4 solutions as well. (It's just for jQuery that steps 5, 6, 7 don't include, but 5, 6, 7 don't include event listeners yet)

File: .bit/tests/sample-solutions/week4/4.4/index.html (hunk 1 above)
Comment (ganning127) [reply .bit/tests/sample-solutions/week4/4.4/index.html]: whoopsies

**Review by emsesc — APPROVED** [review]
thanks 🐱

**Review by blueboxes — APPROVED** [review]

**Pull request merged**